template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Company.WebApplication1.csproj

```diff
@@ -30,6 +31,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.0.0-preview2-final" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Razor.ViewCompilation" Version="2.0.0-preview2-final" PrivateAssets="All" />
     <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="2.0.0-preview2-final" />
+    <PackageReference Include="Microsoft.Composition" Version="1.0.27" PrivateAssets="All" NoWarn="NU1701" Condition="'$(IndividualAuth)' == 'True'" />
```

Comment:
Is this needed for .NET Framework as well? 
AFAIK `AssetTargetFallback` is not a thing for .NET Framework. (atleast `PackageTargetFallback` wasn't)